README.md

```diff
@@ -24,281 +33,98 @@ of applications & services when it comes to rate limiting, authentication, autho
+| Control Plane | The control plane takes the customer desired configuration (declaratively as kubernetes custom resources) as input and ensures all components are configured to obey customer's desired behavior.<br> This repository contains the source code of the kuadrant control plane. |
+| [Kuadrant Operator](https://github.com/Kuadrant/kuadrant-operator) | A Kubernetes Operator to manage the lifecycle of the kuadrant deployment |
+| [Authorino](https://github.com/Kuadrant/authorino) | The AuthN/AuthZ enforcer. As the [external istio authorizer](https://istio.io/latest/docs/tasks/security/authorization/authz-custom/) ([envoy external authorization](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_authz_filter) serving [gRPC](https://grpc.io/) service |
+| [Limitador](https://github.com/Kuadrant/limitador) | The external rate limiting service. It exposes a [gRPC](https://grpc.io/) service implementing the [Envoy Rate Limit protocol (v3)](https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/ratelimit/v3/rls.proto). |
+| [Authorino Operator](https://github.com/Kuadrant/authorino-operator) | A Kubernetes Operator to manage Authorino instances |
+| [Limitador Operator](https://github.com/Kuadrant/limitador-operator) | A Kubernetes Operator to manage Limitador instances |
+
+### Provided APIs
+
+The kuadrant control plane owns the following [Custom Resource Definitions, CRDs](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/):
+
+| CRD | Description |
+| --- | --- |
+| RateLimitPolicy CRD [\[doc\]](/doc/rate-limiting.md) [[reference]](doc/ratelimitpolicy-reference.md) | Enable access control on workloads based on HTTP rate limiting |
+| [AuthPolicy CRD](apis/apim/v1alpha1/authpolicy_types.go) | Enable AuthN and AuthZ based access control on workloads |
+
+Additionally, kuadrant provides the following CRDs
+
+| CRD | Owner | Description |
+| --- | --- | --- |
+| [Kuadrant CRD](https://github.com/Kuadrant/kuadrant-operator/blob/main/api/v1beta1/kuadrant_types.go) | [Kuadrant Operator](https://github.com/Kuadrant/kuadrant-operator) | Represents an instance of kuadrant |
+| [Limitador CRD](doc/ratelimitpolicy-reference.md) | [Limitador Operator](https://github.com/Kuadrant/limitador-operator) | Represents an instance of Limitador |
+| [Authorino CRD](https://github.com/Kuadrant/authorino-operator#the-authorino-custom-resource-definition-crd) | [Authorino Operator](https://github.com/Kuadrant/authorino-operator) | Represents an instance of Authorino |
+| [AuthConfig CRD](https://github.com/Kuadrant/authorino/blob/main/docs/architecture.md#the-authorino-authconfig-custom-resource-definition-crd) | [Authorino](https://github.com/Kuadrant/authorino) | The desired authN and authZ protection for a service |
+
+![architecture](https://i.imgur.com/0vdUZ0l.png)
 
 ## Getting started
 
```

Comment:
Perhaps a one-liner here saying that the steps are for adding RL-only (no auth)?

Reply:
It is supposed to be the "getting started" generic for the kuadrant's protection, not only RL. I will fix that.

Reply:
Fixed